src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -85,6 +90,7 @@ public RunConfiguration() : base(Constants.RunConfigurationSettingsName)
             this.disableAppDomain = false;
             this.disableParallelization = false;
             this.designMode = false;
+            this.shouldCollectSourceInformation = this.designMode;
```

Comment:
nit : Replace `this.shouldCollectSourceInformation = this.designMode;` with `this.shouldCollectSourceInformation = false;`